doc: specify that `longDescription` should be Markdown (#188805)
NixOS/nixpkgs: doc/stdenv/meta.chapter.md

### `longDescription` {#var-meta-longDescription}

> An arbitrarily long description of the package in Markdown format.

Reviewer: ```suggestion
An arbitrarily long description of the package in [CommonMark](https://commonmark.org/) format.
```

> An arbitrarily long description of the package in [CommonMark](https://commonmark.org) Markdown.

Reviewer 2: That makes it look too much like there is a thing called _CommonMark Markdown_, maybe use something like the following instead:

```suggestion
An arbitrarily long description of the package in [CommonMark](https://commonmark.org) Markdown dialect.
```

> An arbitrarily long description of the package in [CommonMark](https://commonmark.org) Markdown.

Reviewer 3: @jtojnar in that case, add "the".

```suggestion
An arbitrarily long description of the package in the [CommonMark](https://commonmark.org) Markdown dialect.
```

> An arbitrarily long description of the package in [CommonMark](https://commonmark.org) Markdown.

Reviewer 3: My latest suggestion was 50% ironic and this is 100% my last suggestion.

> An arbitrarily long description of the package in [CommonMark](https://commonmark.org) Markdown.

Author: I think the sentence is fine as it is and the bikeshed will stay purple.

> An arbitrarily long description of the package in [CommonMark](https://commonmark.org) Markdown.

Reviewer 4: No, no, no, the bikeshed must be blue. Anything else is ridiculous.

> An arbitrarily long description of the package in [CommonMark](https://commonmark.org) Markdown format.

Reviewer 3: ```suggestion
An arbitrarily long description of the package in [CommonMark](https://commonmark.org) Markdown.
```

Either use this suggestion, or add "the".

Reviewer 3: Or to follow pennae's suggestion: Nixpkgs Markdown.
